app/views/home/tos_faq.html.erb

```diff
@@ -285,14 +285,14 @@
       However, there are a number of varieties of works produced by fans that do not fit comfortably into a narrow definition of fanfiction, fanart, vids, or other types of fanworks.  Some of these do fall within our mission.  In particular, original fiction that is part of an Open Doors project is allowed, as are types of original fiction and quasi-original fiction produced within a fandom context.  Examples include such things as anthropomorfic, original fiction that is produced as part of a fandom challenge, exchange, or charity event, and genres such as Original Slash, Original BL, and Regency romances produced in Jane Austen fandom.
       </p>
       <p>
-      At such time as we are able to host art and vids, we anticipate similar policies will apply to those mediums: Art and vids will be considered fannish even in cases where they do not directly depict characters and elements from or use footage from an existing media source if they were created in a fandom context for a fandom audience.  This may include such things as fanart that is intended to be for a particular canon but which does not contain readily identifiable canon characters or elements, art and vids produced as part of a fandom challenge, exchange, or charity event, illustrations of fanfiction, and vids that comment on a particular canon without using any clips from it (e.g., fan-produced videos set in a particular universe).  
+      At such time as we are able to host art and vids, we anticipate similar policies will apply to those mediums: Art and vids will be considered fannish even in cases where they do not directly depict characters and elements from or use footage from an existing media source if they were created in a fandom context for a fandom audience.  This may include such things as fanart that is intended to be for a particular canon but which does not contain readily identifiable canon characters or elements, art and vids produced as part of a fandom challenge, exchange, or charity event, illustrations of fanfiction, and vids that comment on a particular canon without using any clips from it (e.g., fan-produced videos set in a particular universe).
```

Comment:
oooh i see it now. i don't recall deleting any spaces from this line but maybe it happened as i was editing...